sdk/@launchdarkly/observability-dotnet/src/LaunchDarkly.Observability/DefaultNames.cs

```diff
@@ -0,0 +1,9 @@
+namespace LaunchDarkly.Observability
+{
+    internal static class DefaultNames
+    {
+        public const string MeterName = "launchdarkly-plugin-default-metrics";
+        public const string ActivitySourceName = "launchdarkly-plugin-default-activity";
```

Comment:
IIRC the customer has to provide this activity source name if they are emitting their own traces with the System.Diagnostics API for OTEL to pick them up because of the configured activity source? is it confusing then that below we `??` it with the service name? not sure what the semantic meaning of the activity source should be

Reply:
Though the previous SDK didn't actually have a way to emit things against that activity source, so it wouldn't have any effect. With the singleton it actually has a way to be used, so it needs to be safe.

Reply:
(Well, it would still throw without setting the name, but then the name didn't really matter unless someone manually created an activity source with that name).